afl-fuzz.c

```diff
@@ -8071,6 +8094,17 @@ int main(int argc, char** argv) {
 
   if (queue_cur) show_stats();
 
+  /* if we stopped programmatically, we kill the forkserver and the current runner. 
```

Comment:
It looks like most comments start with a capital letter and end with a period. This is also in the [google style guide](https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar).

Reply:
See #23